src/flagStore.ts

```diff
@@ -1,26 +1,25 @@
-import { ConfigurationChangeEvent, commands, window } from 'vscode';
+import { ConfigurationChangeEvent, commands, EventEmitter, window } from 'vscode';
 import InMemoryFeatureStore = require('launchdarkly-node-server-sdk/feature_store');
 import LaunchDarkly = require('launchdarkly-node-server-sdk');
 
-import { debounce } from 'lodash';
+import { debounce, Dictionary, keyBy } from 'lodash';
 
 import { FeatureFlag, FlagConfiguration, FlagWithConfiguration } from './models';
 import { Configuration } from './configuration';
 import { LaunchDarklyAPI } from './api';
 
 const DATA_KIND = { namespace: 'features' };
 
-type FlagUpdateCallback = (flag: FeatureFlag) => void;
+type FlagUpdateCallback = (Object: string) => void;
```

Comment:
What is this change for? Does the callback only accept strings now? (flag key?)

We could probably keep the parameter name the same either way

Reply:
yeah that's all that it's ever done as far as I can understand not sure why typechecking didn't catch it before. Updated so it's `flag` for parameter again